lib/mainnet-staging-short-epoch-genesis.json

```diff
@@ -14440,23 +14440,15 @@
 	"9ddb7f322f6793a3be63ac3f31ac71e7b90d27b76c1cc71761e2c5bf": 1,
 	"4d84219037028f3502c58e293396f9becc8b94ecbf852f404580ef4b": 1,
 
-	"6ea479471d79c2db71e7f5bf55e4b391b310450de81d3b10850eadd2": 1,
-	"691f26be7d3a03d56c40c979c03eed0fc06db5346c47e8d338cc7b11": 1,
-
-	"be2819fd6b1ac86763d5c1de6903b5e8d96f43b23f243e16401f85cb": 1,
-	"44527f7f7b1e49e679de5033f79ec075556adc3e8086194016abdadd": 1
+	"6ea479471d79c2db71e7f5bf55e4b391b310450de81d3b10850eadd2": 1
     },
     "ftsSeed": "76617361206f7061736120736b6f766f726f64612047677572646120626f726f64612070726f766f6461",
     "heavyDelegation": {
 	"a4773a2a8c712bb6dfa47e04ef3b04ab8e415102e1a33886fd2d4084":{"cert":"247ce5fb2a9f7ba82ea084631a66a6151e12dda2d3c885e7e41c43d0c6cdf94d97c0924d95ca236c35c7bfe0874ccd7c9a759ac46d32969dea042345d7b88503","delegatePk":"sc4pa1pAriXO7IzMpByKo4cG90HCFD465Iad284uDYz06dHCqBwMHRukReQ90+TA/vQpj4L1YNaLHI7DS0Z2Vg==","issuerPk":"aN+vWQEX3CndoxLZk3tnSXPb3KYMl4Y5Fq/5VX7ncMHHpedqNdYfVvDvre4JfilQUFph4XnR7nTBz3oD2Is+qw==","omega":0},
 	"9ddb7f322f6793a3be63ac3f31ac71e7b90d27b76c1cc71761e2c5bf":{"cert":"b2f044cf7f0d1aa9c800817436570aabbfa029b3ff9b77a3b83d3205746f176c9fef5aec029aaa5f9c486744bfedb6724c997515ed4f5fdaa2d03700f8708407","delegatePk":"HV5+gKgoYbGoAB34NMkIDvKBrulsRyfCZmS0Fizrl7bUnEfMs9/EyvLcgDaqLdfNLNfFzBSgAR1QWNWADQ0Gig==","issuerPk":"S33PbA1kO5Cad6f47KJcNGW3FcD5IT1zEO2fn3cORj6qKZxT8r9x+Izadz1f+SWx/lufb3z2c7fNiPtHPkwwvA==","omega":0},
 	"4d84219037028f3502c58e293396f9becc8b94ecbf852f404580ef4b":{"cert":"cb4e69ff882552865e93312fd00970bc7ab36296de0c50f50e4a7e0ce4b6cb0eb9a55794e20b670426f35cfbcd5f3368f9de5c403677fafe408e6a92eb5f4a00","delegatePk":"9nyylndB1x0liSdnz0LhpoufvOpEuZ1jJLcrDZYspFk/mj10EkWZUCi/wfiST0hbeunKp2lhx+HbGpNuMlEvEA==","issuerPk":"b0BEmpRyqoitufH0FAJIiTqpaDQYgpJ0uMuiPO3reI1AiAZD5fkvu8o9vWl924+VVwe7UEIRbOGCHk/o6Vel3g==","omega":0},
 
-	"6ea479471d79c2db71e7f5bf55e4b391b310450de81d3b10850eadd2":{"cert":"abc115e9bc3398607558da9ed2433ba00c65ff71e9a625f853374c33d832fc5dc1812dce9c59bd67310def91ffbe44845f45678d1ea9414e832bfa370d21c300","delegatePk":"FVT9+EybM5YFJKmYZoUa0hb6mZEZXmtChhxwpTS/Gn8QeIaDzpFBq889IXTCt5Idew+JT+9f+tiXAluAlCNtng==","issuerPk":"npsfpHUshwIPLCzexfM3PAhQiPhfJ/A6q2X4ZOoON+UJpjhrozM24VQl3PEKKn/jfadzF10Dw7aEWOXVHnN2Vw==","omega":0},
```

Comment:
I can't open original file version for some reason. Are these changes related to some non-mainnet config, right?

Reply:
This genesis is used in mainnet-staging-short-epoch cluster (i. e. staging which is close to mainnet, but with short epoch). These changes exist, because initially there were 7 nodes in the configuration, but later we decided to deploy only 4 (and have only 4 richmen in genesis basically).